git-secrets

```diff
@@ -258,6 +258,8 @@ register_aws() {
   add_config 'secrets.patterns' "${opt_quote}${aws}(ACCOUNT|account|Account)_?(ID|id|Id)?${opt_quote}${connect}${opt_quote}[0-9]{4}\-?[0-9]{4}\-?[0-9]{4}${opt_quote}"
   add_config 'secrets.allowed' 'AKIAIOSFODNN7EXAMPLE'
   add_config 'secrets.allowed' "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+  add_config 'secrets.allowed' "123456789012"
```

Comment:
Yes, this one and the one after seems to be used in all AWS example docs. Funny how it's not `000000000000`. Seems reasonable otherwise